### ISSUE TITLE
Enum annotation for numbers won't be generated 

### ISSUE THREAD

**Issue by Aclrian** [lars-reimann/api-editor#759]
### URL Hash

#/sklearn/sklearn.preprocessing._data/normalize/axis

### Expected Annotation Type

`@enum`

### Expected Annotation Inputs

{0, 1}

**Comment by lars-reimann** [comment]
Duplicate of https://github.com/lars-reimann/api-editor/issues/743.
